cmd/waiter/README.md

```diff
@@ -0,0 +1,25 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+`waiter`
+--------
+
+In a nutshell, it waits until the lock-file is removed. When starting the application a lock-file
+(`--lock-file`) is created, and when the file is removed the `waiter` stops gracefully. When timeout
```

Comment:
since you don't mention the other arguments flags here, I would remove the `(--lock-file)` parenthetical in this line

Reply:
It is helpful to keep sentences and paragraphs on a single line. Hard line breaks create problems with for users searching and maintaining markdown content.

Reply:
Okay, I'm removing the descriptive argument from the paragraph, and also keeping phrases in a single line. Please consider.

Reply:
looks good - marking resolved